tests/data/pato.json

```diff
@@ -78610,7 +78610,7 @@
             "basicPropertyValues": [
               {
                 "pred": "http://www.geneontology.org/formats/oboInOwl#created_by",
-                "val": "cjm"
+                "val": "orcid:1234"
```

Comment:
This should not be fixed (pato.json), but instead _skipped_ by the obographs->sssom parser. No need to fix now, but make an issue to that end.

Reply:
I did a complete find-replace for this substitution.